### PR TITLE
Add `source` property to builds

### PR DESCRIPTION
**Pull request by thgreasi**
Connects To: #350
Change-Type: minor
See: https://github.com/resin-io/resin-ui/issues/897
See: https://github.com/resin-io/resin-api/pull/306#pullrequestreview-41516595
See: https://github.com/resin-io/resin-builder/pull/180

### CODE REVIEW

**Comment by thgreasi** [comment]
There shouldn't be any change in the docs, but is seems that they were not built last time.
Also there were only rejection tests about this method, so nothing changed in the tests.

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -85,11 +85,11 @@ If you feel something is missing, not clear or could be improved, please don't h
...
                 * [.getAll(uuidOrId)](#resin.models.environment-variables.device.getAll) ⇒ <code>Promise</code>
                 * [.getAllByApplication(nameOrId)](#resin.models.environment-variables.device.getAllByApplication) ⇒ <code>Promise</code>
-                * [.create(uuidOrId, name, value)](#resin.models.environment-variables.device.create) ⇒ <code>Promise</code>
+                * [.create(uuidOrId, envVarName, value)](#resin.models.environment-variables.device.create) ⇒ <code>Promise</code>
                 * [.update(id, value)](#resin.models.environment-variables.device.update) ⇒ <code>Promise</code>
                 * [.remove(id)](#resin.models.environment-variables.device.remove) ⇒ <code>Promise</code>
             * [.getAllByApplication(applicationNameOrId)](#resin.models.environment-variables.getAllByApplication) ⇒ <code>Promise</code>
-            * [.create(applicationNameOrId, name, value)](#resin.models.environment-variables.create) ⇒ <code>Promise</code>
+            * [.create(applicationNameOrId, envVarName, value)](#resin.models.environment-variables.create) ⇒ <code>Promise</code>
```
Comment (emirotin) [inline DOCUMENTATION.md]: These changes mean the docs weren't rebuilt last time. Can you please make the build command be `gulp build && npm run docs`?

Comment (thgreasi) [inline DOCUMENTATION.md]: @emirotin changed that, can you take a look and re-review?
Also any idea what should I do with the VB error?

**Comment by emirotin** [comment]
You also have to change the commit message to make the VB happy

**Pull request merged**